doc/submitting.md

```diff
@@ -0,0 +1,119 @@
+
+A full list of game tags, unit IDs and unit names can be found [here](id_reference/game_tags.md).
+
+### Requirements for Models
+
+#### Style
+
+For our replacement graphics, we encourage the "similar but distinct" approach that [OpenTTD](https://www.openttd.org/) used in their project. Their concept is described [here](https://wiki.openttd.org/Graphics_Replacement#Artwork):
+
+> New art is being drawn in the style of the original game, using the original 8bpp palette. The graphics should be a similar but distinct version of the object in question - no graphics [= textures or sprites] may be copied at all from the original. [...] Where possible glitches/bugs in the original graphics should be avoided [...]
+
+In other words, the models should be close to the original, but do not have to be exact copies.
+
+Objects in AoE1 and AoE2 often have a very recognizable shape, size and color. Keep that in mind when you design your model. Textures of buildings should also match the general style of the civilization they are part of. Consistency and recognizability are preferred over historical accuracy.
+
+#### Naming and Structural Conventions
+
+To be compatible with our sprite generation scripts, we require a few (minor) conventions for the `.blend` file.
+
+**Scene**
+
+* Put every mesh object that should be rendered in a Collection with the name `render_me`. This should only contain meshes and not lights, cameras, shadow catchers, etc.
+* Let the model face towards the positive x-axis.
+* Create a camera `dim_camera_45` with rotation (X=60°, Y=0°, Z=45°) and orthographic type, set it as the active camera and fit it to the model. You can do the latter by selecting everything in `render_me` and the camera, then do
+  * View -> Align View -> Align Active Camera to Selected
+
+**Animations**
+
+* Assign an appropriate prefix to the name an **Action** like `move_<something>` or `idle_<something>` (replace `<something>` with a more descriptive string like an index).
+* Remove Actions that are not used in NLA tracks or - if you think they are still useful for others - at least mark them as ununsed by prefixing them with `X_`, e.g. `X_move_<something>`.
+* Our scripts render the sprite by using NLA tracks. Create at least one NLA track for every animation. The naming convention for the NLA track is
+  * `<animation_name>_<index>` in all lowercase and the index starting at 0. Examples: `move_0`, `attack_2`
+
+**Others**
+
+* Please use descriptive names for everything. People who want to build on your work will thank you and the reviewers will be happy, too :) By the way, did you know that you can [add comments in the node editor](https://blender.stackexchange.com/a/32842)?
```

Comment:
Might be worth to use a web archive link here:
`http://web.archive.org/web/20190914174812/https://blender.stackexchange.com/questions/7825/is-there-a-way-to-make-comments-in-the-node-editor/32842`

Reply:
I didn't use the link because it does not directly point to the relevant answer. Looks like this does not work with Web Archive, sadly.